const throwAnError = () => {
    throw new Error('I am an error');
};

export default throwAnError;
